app/services/flow/flows/provider_partner.rb

```diff
@@ -6,13 +6,13 @@ class ProviderPartner < FlowSteps
           path: ->(application) { urls.providers_legal_aid_application_client_has_partner_path(application) },
           forward: lambda do |_application, options|
             if options[:has_partner]
-              :partners_details
+              :details
             else
               :check_provider_answers
             end
           end,
         },
-        partners_details: {
+        details: {
```

Comment:
I guess we might run into this limitation (i.e "flow step" clashes) at some point. I'll add it as a point of consideration to the existing tech debt ticket 👍

Reply:
Been doing some digging and with play with the `prefix_step_with :partner` controller setting.

It _looks_ as though it will allow the controller to be namespaces as `Partners::Details` but be flowable as `partner_details`.

I've not had cause to use it before but I guess it's exactly for this circumstance :+1: